.editorconfig

```diff
@@ -9,6 +9,14 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.java]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Recommend just enabling Spotless instead.

Reply:
Spotless is now enabled. I will keep these basic formatting rules here for all the IDEs